packages/components/src/global/scale.css

```diff
@@ -43,6 +43,16 @@
   --scl-z-index-50: 50;
   --scl-z-index-60: 60;
   --scl-z-index-70: 70;
+  /* temporary hard coded values, these will be added to design-tokens package */
+  --telekom-color-functional-focus-standard: #7d94f9;
+  --telekom-color-functional-focus-inverted: #2238df;
+  --telekom-color-functional-focus-on-dark: #7d94f9;
+}
+
+[data-mode='dark'] {
+  --telekom-color-functional-focus-standard: #2238df;
+  --telekom-color-functional-focus-inverted: #7d94f9;
+  --telekom-color-functional-focus-on-dark: #7d94f9;
```

Comment:
`inverted` and `standard` are swapped here 🙃 — they should go like this:

light mode:
```
--telekom-color-functional-focus-standard: #2238df;
--telekom-color-functional-focus-inverted: #7d94f9;
--telekom-color-functional-focus-on-light-background: #2238df;
--telekom-color-functional-focus-on-dark-background: #7d94f9;
``` 

dark mode:
```
--telekom-color-functional-focus-standard: #7d94f9;
--telekom-color-functional-focus-inverted: #2238df;
``` 

Also, we agreed to rename `…on-dark` to `…on-dark-background` 🙏